ADD  -- Task 12(G): Creación de la entidad oferta

marvicmar opened the pull request.

marvicmar committed:
Add entity Offer

marvicmar committed:
Add entity Offer

alepervaz reviewed (approved):
el código implementado esta completo y desarrollado de forma correcta

alesanfe reviewed (changes requested):
Las restricciones están mal, el máximo está incluido,

marvicmar committed:
Update Offer.java

alesanfe committed:
fix(init): Add rol student

Formato de los comentarios sigue el formato correspondiente al resto de las entidades del trabajo.

alepervaz reviewed (changes requested):
faltan getter,setter y entity

marvicmar committed:
Update Offer.java

Add @Getter, @Setter y @Entity

alepervaz reviewed (approved):
todo correcto
